src/airflow/dags/common_airflow.py

```diff
@@ -45,7 +46,7 @@
 # Shared DAG construction parameters.
 shared_dag_args = dict(
     owner="Open Targets Data Team",
-    retries=3,
+    retries=1,
```

Comment:
BTW, the reason for this change: when using PySpark + Dataproc, in my experience, 99% of errors are due to mistakes in the code and not some transient failures. In this case, restarting the step 3 times just wastes time & money, instead of failing immediately and allowing to start investigations as to the cause of failure